--- anyremote-6.7.3/Makefile.am~	2019-11-08 12:38:58.000000000 +0100
+++ anyremote-6.7.3/Makefile.am	2019-11-08 12:38:58.000000000 +0100
@@ -27,7 +27,6 @@
 	test -z $(DESTDIR)$(docdir) || mkdir -p -- . $(DESTDIR)$(docdir)
 	cp -r cfg-data $(DESTDIR)$(pkgdatadir)
 	rm -rf $(DESTDIR)$(pkgdatadir)/cfg-data/obsoleted
-	cp -r doc-html ChangeLog README COPYING AUTHORS $(DESTDIR)$(docdir)/
 	find $(DESTDIR)$(pkgdatadir) $(DESTDIR)$(docdir) -type f -exec chmod 644 {} \;
 	chmod a+x $(DESTDIR)$(pkgdatadir)/cfg-data/Utils/all-in-1.py 
 	chmod a+x $(DESTDIR)$(pkgdatadir)/cfg-data/Utils/generate_cover.sh
