--- anyremote-5.0/configure.in~	2009-06-10 15:08:38.000000000 +0000
+++ anyremote-5.0/configure.in	2009-10-27 11:25:59.000000000 +0000
@@ -36,7 +36,7 @@
 AC_HEADER_STDC
 AC_HEADER_SYS_WAIT
 
-CFLAGS="$CFLAGS -I/usr/local/include -Wall -D_REENTRANT -O2 -g"
+CFLAGS="$CFLAGS -I/usr/include -Wall -D_REENTRANT"
 
 AC_CHECK_HEADERS([fcntl.h stdlib.h string.h sys/ioctl.h sys/time.h termios.h unistd.h glib.h])
 
